cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/EntityOperatorReconcilerTest.java

```diff
@@ -87,6 +91,9 @@ public void reconcileWithToAndUo(VertxTestContext context) {
         ArgumentCaptor<Role> operatorRoleCaptor = ArgumentCaptor.forClass(Role.class);
         when(mockRoleOps.reconcile(any(), eq(NAMESPACE), eq(KafkaResources.entityOperatorDeploymentName(NAME)), operatorRoleCaptor.capture())).thenReturn(Future.succeededFuture());
 
+        ArgumentCaptor<NetworkPolicy> netPolicyCaptor = ArgumentCaptor.forClass(NetworkPolicy.class);
+        when(mockNetPolicyOps.reconcile(any(), eq(NAMESPACE), eq(CruiseControlResources.networkPolicyName(NAME)), netPolicyCaptor.capture())).thenReturn(Future.succeededFuture());
```

Comment:
You have a wrong name here. Maybe that is why the tests failed.

Reply:
``` when(mockNetPolicyOps.reconcile(any(), eq(NAMESPACE), eq(KafkaExporterResources.deploymentName(NAME)), netPolicyCaptor.capture())).thenReturn(Future.succeededFuture());```

Is this good..
But still getting error
`Cannot invoke "io.vertx.core.Future.map(Object)" because the return value of "io.strimzi.operator.common.operator.resource.NetworkPolicyOperator.reconcile(io.strimzi.operator.common.Reconciliation, String, String, io.fabric8.kubernetes.api.model.HasMetadata)" is null`

Reply:
You have to doublecheck you have the parameters (the names) correctly. In the example above you use `KafkaExporterResources.deploymentName(NAME)` => that is wrong for the Entity Operator. `KafkaResources.entityOperatorDeploymentName` might be the right one here? The same seems to apply to the other tests there.

Reply:
Thanks for it. I Had tried that before but it didn't resolve because the error was due to other things
I have fixed it now.